docs/setup-cluster/slurm/slurm-requirements.rst

```diff
@@ -228,6 +228,94 @@ interacts with PBS, we recommend the following steps:
+   utilize a single GPU on each node. To fully utilize multiple GPUs, you must either manually
+   define ``CUDA_VISIBLE_DEVICES`` appropriately or provide the ``pbs.slots_per_node`` setting in
+   your experiment configuration to indicate how many GPU slots are intended for Determined to use.
+
+-  Ensure ``ngpus`` resource is defined with appropriate values.
+
+   It is necessary to define the ``ngpus`` resource value for every node on the cluster for
+   DeterminedAI to work successfully. Additionally, the resource should also have appropriate flags
+   for PBS to process the value properly while scheduling jobs. Use the following command to check
+   if ``ngpus`` resource is defined with appropriate flags.
+
+   .. code:: bash
+
+      [~]$ qmgr -c "list resource ngpus"
+      Resource ngpus
+          type = long
+          flag = hn
+
+   The above output indicates that ngpus is defined as a type ``long`` resource with flags ``h``
+   (indicating this is a host level resource) and ``n`` (indicating this is a consumable resource).
+   If the nodes on your hpc cluster do not provide the same output, use the following commands to
+   create and set the resource ``ngpus`` for every node.
+
+   .. code:: bash
+
+      [~]$ qmgr -c "create resource ngpus type=long, flag=hn" # create the ngpus resource
+      [~]$ qmgr -c "set node <nodename> ngpus=<number of GPUs>" # set the value for ngpus
+
+   If you are using virtual nodes (vnodes), ensure ``ngpus`` value is only set on the vnodes and not
+   the actual parent node. Below are the commands and sample output to ensure the same.
+
+   .. code:: bash
+
+      [~]$ sudo qmgr -c "list node node002[0] resources_available"
+      Node node002[0]
+          resources_available.arch = linux
+          resources_available.host = node002
+          resources_available.hpmem = 0b
+          resources_available.mem = 45943mb
+          resources_available.ncpus = 18
+          resources_available.ngpus = 2 # ngpus value is set on vnode node002[0]
+          resources_available.vmem = 46933mb
+          resources_available.vnode = node002[0]
+      [~]$ sudo qmgr -c "list node node002 resources_available"
+      Node node002
+          resources_available.accel_type = tesla
+          resources_available.arch = linux
+          resources_available.host = node002
+          resources_available.hpmem = 0b
+          resources_available.mem = 0b
+          resources_available.ncpus = 0 # ngpus value is not set on parent node node002
+          resources_available.Qlist = gpuQ,gpu_hi_priQ
+          resources_available.vmem = 0b
+          resources_available.vnode = node002
+
+   If the ``ngpus`` value is set on the parent node, use the below command to unset it.
+
+   .. code:: bash
+
+      [madagund@node003 ~]$ sudo qmgr -c "unset node <node_name> resources_available.ngpus"
+
+   The next step is to ensure that ``ngpus`` is listed as a resource in the
+   ``<sched_priv_directory>/sched_config`` file.
+
```

Comment:
Next, make sure that ``ngpus`` is listed as a resource in the ``<sched_priv_directory>/sched_config`` file.